packages/react/src/file-input/FileInput.tsx

```diff
@@ -93,70 +138,100 @@ export const FileInput = forwardRef<HTMLInputElement, FileInputProps>(
 		const fallbackHint =
 			hint || (accept && `Files accepted: ${acceptedFileExtensions}`);
 
+		const buttonLabel = `Select ${multiple ? 'files' : 'file'}`;
```

Comment:
This was my doing I think, but I see we use `const fileOrFiles = multiple ? 'files' : 'file';` in `getSelectedFilesMessage()`, so we should probably just create the var here and pass the result to this and the function

Reply:
Sorted